src/pages/index.js

```diff
@@ -1,7 +1,9 @@
+import { Link } from 'gatsby';
 import React from 'react';
 import Image from '../components/image';
 import Layout from '../components/layout';
 import SEO from '../components/seo';
+import 'antd/dist/antd.css';
```

Comment:
remove from index.js, you could try it in layout.js before the index.styl instead